Guidelines.md

```diff
@@ -14,24 +14,23 @@ For example, `Cake.Email` is the name that clearly identifies the addin for Cake
 
 ## NuGet Package Icon
 
-Addins should use the Cake Contrib icon rather than the Cake icon. You can do so by adding the following line in your addin's `.csproj`:
-
+Addins should use the Cake Contrib icon rather than the Cake icon or any other custom icon. Also, addins should embed the icon in their nuget package rather than rely on an external web site to host their icon. You can do so by adding the following line in your addin's `.csproj`:
 ```xml
-<PackageIconUrl>https://cdn.jsdelivr.net/gh/cake-contrib/graphics/png/cake-contrib-medium.png</PackageIconUrl>
+<PackageIcon>MyIconFile.png</PackageIcon>
```

Comment:
Wouldn't it make sense to provide a better default than `MyIconFile.png`, since probably a few people are just copy/pasting from here?

Reply:
I have no objection to provide a better default. Suggestions?

I used `PackageIcon.png` in my own addins such as [Cake.Email.Common](https://github.com/cake-contrib/Cake.Email.Common/blob/develop/Source/Cake.Email.Common/Cake.Email.Common.csproj#L17) for example. Does that sound reasonable?

Reply:
@Jericho Sounds good to me

Reply:
Fixed